lib/season_stats.rb

```diff
@@ -32,11 +32,23 @@ def worst_coach
   end
 
   def most_accurate_team
-
+    teams_grouped = @game_teams.group_by(&:team_id)
```

Comment:
Needs to be filtered by season, which will be a method I don't have the brainpower for tonight. Not ready for merging